presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -7442,13 +7442,14 @@ private void testTimestampPrecisionInsert(Session session, HiveStorageFormat sto
             return;
         }
 
-        String createTable = "CREATE TABLE test_timestamp_precision (ts TIMESTAMP) WITH (format = '%s')";
-        @Language("SQL") String insert = "INSERT INTO test_timestamp_precision VALUES (TIMESTAMP '%s')";
+        String createTable = "CREATE TABLE test_timestamp_precision_insert (ts TIMESTAMP) WITH (format = '%s')";
```

Comment:
We typically are using `io.prestosql.testing.sql.TestTable#randomTableSuffix`

Reply:
fixed and pushed